assets/pages/contributions-thankyou/monthlyContributionsPending.jsx

```diff
@@ -0,0 +1,68 @@
+// @flow
```

Comment:
Will the Pending and Thank you page share the same design? I would reconsider call the CSS classes thank-you, that can lead to confusion and side effects in the future. a.k.a. someone change the Thank You style and the pending page also changes.